output/xml/default.xml

```diff
@@ -132,6 +132,7 @@
     <property name="context_menu" type="bool" help="Generates event handler for displaying of menu assigned to this widgets as a context menu.">1</property>
     <property name="context_help" type="wxString_i18n" help="Attach context-sensitive help to the window. Note: The Project's &quot;help_provider&quot; property must be set for context-sensitive help to work."/>
     <property name="enabled" type="bool" help="Enable or disable the window for user input. Note that when a parent window is disabled, all of its children are disabled as well and they are reenabled again when the parent is.">1</property>
+    <property name="create_children" type="bool" help="Automatically create children when class instansiated. c++ only">1</property>
```

Comment:
Ignoring the typographic errors i don't think this property belongs here. This form of two-step initialization can only be used for toplevel widgets, because these are the only ones the user can use directly. If this gets enabled for child elements it will break the code because the generated code doesn't know anything about this initialization type and will always use the one-step initialization.

Why is this limited to C++ only? Shouldn't it be available for the other languages too?